internal/elasticsearch/client.go

```diff
@@ -284,7 +286,7 @@ func WaitForIndices() (string, error) {
 	catIndices := func() error {
 		r := curl.HTTPRequest{
 			URL:               "http://localhost:9200/_cat/indices?v",
```

Comment:
Should we update the reference to the elasticsearch URL here too?

Reply:
Yes! Good catch